docs/src/mdsvex/h1.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
+  import { onMount } from 'svelte';
+
+  export let id = null;
+
+  let href = id && `#${id}`;
+  onMount(function createSapperCompatiblePermalink() {
+    if (id == null) {
+      return;
+    }
+
+    href = window.location.origin + window.location.pathname + href;
+  });
+</script>
+
+<h1 {id}>
+  <slot />
+  {#if href != null}
+    <a {href} class="ml">#</a>
+  {/if}
```

Comment:
Don't you think it will look better on the left?

Reply:
Nah. It might have been better on the left if that left was absolutely positioned (i.e. not shift the heading to the right) + appeared only on hover, but as of now, I think this works nicely too